src/beacon.rs

```diff
@@ -16,6 +16,8 @@
 // See the Licences for the specific language governing permissions and limitations relating to
 // use of the MaidSafe Software.
 
+#![deny(missing_docs)]
```

Comment:
Sorry @benjaminbollen my bad - can we switch this on in the `lib.rs` file instead